Changelog.md

```diff
@@ -5,6 +5,7 @@
 ### Changed
 - **Breaking Change** Minimum version of `RxSwift` is now 5.0. [#1846](https://github.com/Moya/Moya/pull/1846) by [@LucianoPAlmeida](https://github.com/LucianoPAlmeida).
 - **Breaking Change** Minimum version of `ReactiveSwift` is now 6.0. [#1849](https://github.com/Moya/Moya/pull/1849) by [@sunshinejr](https://github.com/sunshinejr).
+- **Breaking Change** Drop `Result`. [#1858](https://github.com/Moya/Moya/pull/1858) by [@larryonoff](https://github.com/larryonoff).
```

Comment:
Can we be a little bit more verbose on this one? "Removed `Result` dependency in favor of Swift 5 Result type" or something along the lines.